tests/contracts/main/application/conftest.py

```diff
@@ -0,0 +1,59 @@
+"""
+This file is part of nucypher.
+
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+
+import pytest
+
+
+@pytest.fixture()
+def token(deploy_contract, token_economics):
+    # Create an ERC20 token
+    token, _ = deploy_contract('TToken', _totalSupplyOfTokens=token_economics.erc20_total_supply)
+    return token
+
+
+@pytest.fixture()
+def threshold_staking(deploy_contract):
+    threshold_staking, _ = deploy_contract('ThresholdStakingForPREApplicationMock')
+    return threshold_staking
+
+
+@pytest.fixture()
+def pre_application(testerchain, token, threshold_staking, deploy_contract, token_economics):
+    reward_duration = 60 * 60
+    deauthorization_duration = 60 * 60
+    min_authorization = token_economics.minimum_allowed_locked
+    min_worker_seconds = 24 * 60 * 60
+
+    # Creator deploys the PRE application
+    contract, _ = deploy_contract(
```

Comment:
```suggestion
    contract, _receipt = deploy_contract(
```

In tests and fixtures, I'd much prefer to use names even for objects which are not subsequently relevant, for legibility purposes.